front-end/src/components/aptdetails/Notes.jsx

```diff
@@ -46,10 +45,10 @@ const Notes = ({ apartmentData }) => {
   };
   return (
     <React.Fragment>
-      <Card sx={{ height: 350, textAlign: "center" }}>
+      <Card sx={{ height: 350, textAlign: "center", border: 0.5 }}>
         <Typography
-          className=" text-white text-center py-3 text-lg font-semibold rounded-t-lg"
-          bgcolor={"#206129"}
+          className="text-center py-3 text-lg font-semibold"
+          bgcolor={"#e3e7d3"}
         >
           Notes
         </Typography>
```

Comment:
Make sure the margin top I added on line 56/57 for the button gets fixed if you have to solve a conflict regarding it.